README.md

```diff
@@ -7,95 +7,93 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## PHP
 
 1. All PHP code MUST adhere to the [PSR-2 Coding Standard](http://www.php-fig.org/psr/psr-2/) (which MUST follow the [PSR-1 Coding Standard](http://www.php-fig.org/psr/psr-1/)).
-2. Short array syntax (`[]` instead of `array()`) MUST be used.
-3. All PHP code MUST adhere to the [PSR-4 Autoloading](http://www.php-fig.org/psr/psr-4/) Standard.
-4. Class names MUST be declared in UpperCamelCase.
-  1. Acronyms MUST be treated as normal words. `HttpException`, not `HTTPException`.
-5. Variable names MUST be declared in camelCase.
-  1. Acronyms MUST be treated as normal words. `xmlHttpRequest` not `XMLHTTPRequest`.
-6. Scripts MUST be named in camelCase describing their primary function.
-7. Non-static functions MUST NOT be called statically.
-8. Static functions MUST NOT be called non-statically.
-9. Comments SHOULD be used to provide explanation for “why” rather than “how” and SHOULD be used when there isn’t a way to make the code simpler or self-documenting.
-10. Nested ternary operators MUST NOT be used.
-11. PHP file encoding must be UTF-8.
-12. `if` statements which span multiple lines should be split as follows:
-
-  ```php
-  if ($longVariableNameNumber1
-      && $longVariableNameNumber2
-      && $longVariableNameNumber3
-      && $longVariableNameNumber4
-      || $longVariableNameNumber5) {
-    // body of control structure
-  }
-  ```
-
-13. There MUST NOT be any assignment or modification of variables in the expressions of `if` statements.
-
-  ```php
-  if ($response = $request->getResponse()) {
-    // not allowed
-  }
-  ```
-14. Abstract class names MUST begin with `Abstract`.
-
-  ```php
-  abstract class AbstractGenerator {
-  }
-  ```
-15. Interface names MUST end with `Interface`.
-
-  ```php
-  interface GeneratorInterface {
-  }
-  ```
-16. 'Not' logical operators MUST NOT have whitespace between them and the subject being negated.
-  ```php
-  $true = !false;
-  ```
+1. Short array syntax (`[]` instead of `array()`) MUST be used.
+1. All PHP code MUST adhere to the [PSR-4 Autoloading](http://www.php-fig.org/psr/psr-4/) Standard.
+1. Class names MUST be declared in UpperCamelCase.
+    1. Acronyms MUST be treated as normal words. `HttpException`, not `HTTPException`.
+1. Variable names MUST be declared in camelCase.
+    1. Acronyms MUST be treated as normal words. `xmlHttpRequest` not `XMLHTTPRequest`.
+1. Scripts MUST be named in camelCase describing their primary function.
+1. Non-static functions MUST NOT be called statically.
+1. Static functions MUST NOT be called non-statically.
+1. Comments SHOULD be used to provide explanation for “why” rather than “how” and SHOULD be used when there isn’t a way to make the code simpler or self-documenting.
+1. Nested ternary operators MUST NOT be used.
+1. PHP file encoding must be UTF-8.
+1. `if` statements which span multiple lines should be split as follows:
+    ```php
```

Comment:
Could you reinstate the newline above the code block? Not having it breaks the renderer I'm using (MacDown).